examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Owner.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<p>TYPO: <code>DIABETES</code>  -&gt; <code>HEART</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>This is saying for people who died of heart disease lets check their DIABETES column mean. I don't think this is a typo.</p>